test/tests/pbs_smoketest.py

```diff
@@ -1458,3 +1458,20 @@ def test_interactive_job(self):
         self.server.expect(JOB, {'job_state': 'R'}, id=jid)
         self.server.delete(jid)
         self.server.expect(JOB, 'queue', op=UNSET, id=jid)
+
+    def test_man_pages(self):
+        """
+        Test basic functionality of man pages
+        """
+        pbs_conf = self.du.parse_pbs_config(self.server.shortname)
+        manpath = os.path.join(pbs_conf['PBS_EXEC'], "share", "man")
+        pbs_cmnds = ["pbsnodes", "qsub"]
+        for pbs_cmd in pbs_cmnds:
+            cmd = ["export MANPATH=$MANPATH:%s;" % manpath, "man %s" % pbs_cmd]
```

Comment:
This can be a little more concise.  If you put VAR=VAL in front of a command, it will be added to the environment for that command only.  You can do this all in one command instead of two.  I think you'll still need as_script=True.  The other option is to directly add MANPATH do the environment via os.environ() before running the command.  This way you'll only do it once.